source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/WNBCalculationOptions.java

```diff
@@ -85,6 +87,14 @@ public void setUseReceptorHeights(final boolean useReceptorHeights) {
     this.useReceptorHeights = useReceptorHeights;
   }
 
+  public boolean isSubReceptorsEnabled() {
```

Comment:
Not the normal getter name I would expect. Doesn't this give issues when serializing to/from json?

Reply:
Yes might be better to keep it in sync.